helm-chart/example-configurations/minimal-values.tmpl

```diff
@@ -1,5 +1,5 @@
 #
-# Basic configuration file for Renku 0.7.1
+# Basic configuration file for Renku 0.7.8
```

Comment:
I think we should just get rid of the version tag here, no way we'll always remember to bump this.

Reply:
Yes, I agree. I've removed it